gui/gui-manager.cpp

```diff
@@ -517,7 +517,6 @@ void GuiManager::runLoop() {
 		if (iconsChanged) {
 			for (DialogStack::size_type i = 0; i < _dialogStack.size(); ++i) {
 				setTarget(_dialogStack[i]);
-				sendCommand(kIconsSetLoadedCmd, 0);
 			}
```

Comment:
I would suggest to comment out the whole `for` block here. The `setTarget()` is useless without the `sendCommand()`.
Since this is a temporary workaround, I think I slightly prefer comment it out rather than removing the lines.